kirovy/services/cnc_gen_2_services.py

```diff
@@ -98,13 +98,13 @@ class ErrorMsg(enum.StrEnum):
         CORRUPT_MAP = _("Could not parse map file.")
         MISSING_INI = _("Missing necessary INI sections.")
 
-    def __init__(self, uploaded_file: UploadedFile | File):
+    def __init__(self, uploaded_file: UploadedFile | File, *, ini_section_check: bool = True):
```

Comment:
Todo: remove this jank and make a separate way to easily spin up the ini handler